proposals/20220919-airflow_orchestration.md

```diff
@@ -0,0 +1,35 @@
+#### SIG TFX-Addons
+# Project Proposal
+
+**Your name:** Varun Murthy
+
+**Your email:** murthyvs@google.com
+
+**Your company/organization:** Google Core ML TFX Team (MTV + Seoul)
+
+**Project name:** Airflow Orchestration
+
+## Project Description
+Moving Airflow to TFX add-ons due to decreasing native support.
+
+## Project Category
+Orchestrator
+
+## Project Use-Case(s)
+We are moving Airflow from tfx/orchestration to tfx-addons. Native support for Airflow won't be provided in the near future.
```

Comment:
A use-case describes how the project will be used by developers.  In this case it should be something more like:

> Apache Airflow is a widely used orchestrator for ML workflows.  Developers can use Airflow with TFX to orchestrate TFX pipelines.  This project continues support for Airflow, which includes the web console and CLI tooling which developers can use to monitor and control their TFX pipelines.

Reply:
Imagine that you're a developer who is new to TFX.  What do you need to know about Airflow, and how it fits with TFX?  Why should you consider installing this module?